<?xml version="1.0" ?>

<launch>    

    <param name="gazebo"                   value="true"     type="bool"/>
    <param name="gazebo_robot_name"        value="diana_soccer" />
    
    <param name="offset_file_path"  value="$(find heroehs_manager)/config/offset.yaml"/>	
    <param name="robot_file_path"   value="$(find heroehs_manager)/config/robot.robot"/>
    <param name="init_file_path"    value="$(find heroehs_manager)/config/dxl_init.yaml"/>   
    	
    <node name="offset_tuner_server" pkg="offset_tuner_server" type="offset_tuner_server" output="screen">
    </node>
    
</launch>
